### PR TITLE
New version: Tetrate.func-e version 1.1.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41730)

### CODE REVIEW

File: manifests/t/Tetrate/func-e/1.1.2/Tetrate.func-e.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Tetrate.func-e
+PackageVersion: 1.1.2
+PackageLocale: en-US
+Publisher: Tetrate
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author:
```
Comment (Trenly) [inline manifests/t/Tetrate/func-e/1.1.2/Tetrate.func-e.locale.en-US.yaml:11]:
```suggestion
PublisherUrl: https://github.com/tetratelabs
PublisherSupportUrl: https://github.com/tetratelabs/func-e/issues
# PrivacyUrl: 
Author: codefromthecrypt, liamawhite, and contrubutors
```

File: manifests/t/Tetrate/func-e/1.1.2/Tetrate.func-e.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ShortDescription: func-e (pronounced funky) makes running Envoy easy
+Description: func-e (pronounced funky) allows you to quickly see available versions of Envoy and try them out. This makes it easy to validate configuration you would use in production. Each time you end a run, a snapshot of runtime state is taken on your behalf. This makes knowledge sharing and troubleshooting easier, especially when upgrading. Try it out!
+Moniker: func-e
+Tags:
+- envoy
+- func-e
+- tetrate
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl:
```
Comment (Trenly) [inline manifests/t/Tetrate/func-e/1.1.2/Tetrate.func-e.locale.en-US.yaml:27]:
```suggestion
ReleaseNotes: func-e 1.1.2 contains only version updates. This release is only to inform people that Envoy 1.21 is out, and that is the first version that allows M1 Macs to run without emulation.
ReleaseNotesUrl: https://github.com/tetratelabs/func-e/releases/tag/v1.1.2
```

**Comment by codefromthecrypt** [comment]
@vedantmgoyal2009 thanks for this.. does this mean we no longer need to push pull requests here to do version bumps?

**Comment by vedantmgoyal2009** [comment]
> @vedantmgoyal2009 thanks for this.. does this mean we no longer need to push pull requests here to do version bumps?

Yeah! I will also be making some changes so that the automation can fill `ReleaseNotes` and `ReleaseNotesUrl`.

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**